sdk/communication/communication-chat/README.md

```diff
@@ -136,7 +136,7 @@ for await (const chatMessage of chatThreadClient.listMessages()) {
 
 Once a thread is created, you can then add and remove users from that thread. By adding users, you give them access to be able to send messages to the thread.
 You will need to start by getting a new access token and identity for that user. The user will need that access token in order to initialize their chat client.
-More information on tokens here: [Authenticate to Azure Communication Services](https://review.docs.microsoft.com/azure/project-spool/concepts/authentication?branch=pr-en-us-104477&tabs=javascript)
+More information on tokens here: [Authenticate to Azure Communication Services](https://docs.microsoft.com/azure/communication-services/concepts/authentication?tabs=javascript)
```

Comment:
While you're at it, can you fix the Next Steps section? The checklist thing doesn't render, let's remove it

![image](https://user-images.githubusercontent.com/1885005/94493238-f0b94180-01a0-11eb-81de-d08ce361588f.png)